docker-build/usage/metadata-ghcr.md

```diff
@@ -0,0 +1,26 @@
+#### Tag Metadata and GHCR
```

Comment:
Probably worth documenting how to write a custom usage example to fit the heading flow of the "parent" README. 

Is it with enforcing somehow with some sort of validation check? Maybe something for the future.